Localize/lang/strings.json

```
@@ -700,6 +697,7 @@
   "W070M2": "of {max}",
   "W6FdMh": "Required. The name of the new property.",
   "WGwH45": "Clear",
+  "WK6hZX": "Select all text in {label}",
   "WLj8DH": "Please enter a valid dictionary",
   "WOOYW1": "Actions perform operations on data or communicate between systems to execute.",
```

"systems to execute"?

```suggestion
  "WOOYW1": "Actions perform operations on data, communicate between systems, or run other tasks.",
```